hypertension/vro-main/test/main_test.py

```diff
@@ -1,12 +1,18 @@
-from lib.main import main
-
+from lib.main import main, get_pdf
 from test.doubles.aws_secrets_manager import mock_get_lighthouse_rsa_key
 
 
+test_event = {"html": "<html><body>Hello World!</body></html>"}
+
+
 def test_main():
-    assert main() == {
+    assert main(test_event) == {
         "statusCode": 200,
         "body": mock_get_lighthouse_rsa_key(
             "Fake. Doesnt Matter. Just need something here."
         ),
     }
+
+
+def test_get_pdf():
+    assert isinstance(get_pdf(test_event), bytes) is True
```

Comment:
Thanks for adding some tests!

Seems to me it could be difficult to do more than these `isinstance` tests, but I figure I should ask if you thought of anything.

Reply:
I debated adding a dependency to help test this, and @tadhg-ohiggins thought that would be fine, so this is updated to verify the output with pdfminer (and I have it in the other pdf_generator file): https://github.com/department-of-veterans-affairs/Virtual-Regional-Office/pull/15/commits/2c503171846b03dc6571e1a9916630fb30968fa1#diff-3648e22824e461bf03a66d761604d8f4d3e1db6853c4d736bb2d01f1a1589d99L1